scrapers/errata.py

```diff
@@ -69,6 +69,53 @@ def update_neo4j(self, advisories):
                 'update_date': advisory['update_date'],
                 'updated_at': advisory['updated_at']
             })[0]
+            container_adv = False
+
+            for associated_build in self.get_associated_builds(advisory['id']):
+                # Even if a node has two labels in the database, Neo4j returns the node
+                # only with the specific label you asked for. Hence we check for labels
+                # ContainerKojiBuild and KojiBuild separately for the same node.
+                build = ContainerKojiBuild.nodes.get_or_none(id_=associated_build['id_'])
+                if not build:
+                    build = KojiBuild.nodes.get_or_none(id_=associated_build['id_'])
+
+                if build and not container_adv:
+                    if build.__label__ == 'ContainerKojiBuild':
+                        adv.add_label(ContainerBuildAdvisory.__label__)
+                        container_adv = True
+
+                # If this is set, that means it was once part of the advisory but not anymore.
+                # This relationship needs to be deleted if it exists.
+                if associated_build['removed_index_id']:
+                    if build:
+                        adv.attached_builds.disconnect(build)
+                else:
+                    # This key shouldn't be stored in Neo4j
+                    del associated_build['removed_index_id']
+                    # Query Teiid and create the entry only if the build is not present in Neo4j
+                    if not build:
+                        attached_build = self.get_koji_build(associated_build['id_'])
+                        if attached_build:
+                            if self.is_container_build(attached_build):
+                                build = ContainerKojiBuild.get_or_create(
+                                    {'id_': associated_build['id_']})[0]
+                            else:
+                                build = KojiBuild.get_or_create(
+                                    {'id_': associated_build['id_']})[0]
+
+                    # This will happen only if we do not find the build we are looking for in Teiid
+                    # which shouldn't usually happen under normal conditions
+                    if not build:
+                        log.info('Build id {} was not found in Teiid!'.format(
+                            associated_build['id_']))
+                        continue
+
+                    if (adv.__label__ != ContainerBuildAdvisory.__label__ and
+                            build.__label__ == 'ContainerKojiBuild'):
```

Comment:
I'd say either change `'ContainerKojiBuild'` to be `ContainerKojiBuild.__label__` or replace `ContainerBuildAdvisory.__label__` to be `'ContainerBuildAdvisory'` . Two different ways is confusing to the reader.